pages/blog/introducing-envelop.mdx

```diff
@@ -0,0 +1,310 @@
+
+**[You can read more about sharing/composing envelops here](https://www.envelop.dev/docs/composing-envelop)**
+
+## "Babel for GraphQL" - New Features for the GraphQL Engine
+
+Since we allow developers to take part in any phase of the execution, it means that you can easily add new features for the GraphQL engine, and not just features that come on top of GraphQL.
+
+For example, one of the Envelop plugins ([`useExtendedValidation`](https://www.envelop.dev/plugins/use-extended-validation)) allows developers now to write and run GraphQL validations, with access to the operation variables. That means you can write simple validations now without making it part of your schema.
+
+One of the things that is also possible now is `@oneOf` - a spec suggestion that is still in discussion for adding input unions, but already available for you if you use Envelop, because extended validations can access variables and can do additional things that was difficult to do before.
+
+Here are some additional examples:
+
+- `perEventContextResolver`: suggested in [this PR](https://github.com/graphql/graphql-js/pull/2485), and almost [available in envelop](https://github.com/dotansimha/envelop/pull/183).
+- `@oneOf`: suggested in [this PR](https://github.com/graphql/graphql-spec/pull/825), and now [available in envelop](https://github.com/dotansimha/envelop/blob/main/packages/plugins/extended-validation/README.md#union-inputs-oneof).
+
+## Adoption and Migration Path / Framework for Frameworks
+
+If you are already using GraphQL, you are probably using a server that comes with all the features built-in. This is great in some cases, but if you wish to have that extra flexibility, you can migrate to Envelop. You can even use Envelop with other server frameworks without migrating the entire pipeline (see examples section below).
+
+GraphQL is also widely adopted in the JAMStack world - and libraries that offer GraphQL out of the box migrate to Envelop to simplify parts of the code, and to allow their users to extend the GraphQL layer in a simple way.
+
+[Redwood](https://redwoodjs.com/) is a great example. We start with a small suggestion PR, and the Redwood team was open for new ideas - so [now you can use envelop if you are a Redwood user](https://community.redwoodjs.com/t/migrate-your-redwood-graphql-api-with-graphql-helix-and-envelop/2264) !
+
+[Here is a thread about why Redwood now gives you the option](https://community.redwoodjs.com/t/using-graphql-envelop-helix-in-redwood-v0-35/2276) to replace Apollo Server with [GraphQL-Helix](https://github.com/contrawork/graphql-helix) + `Envelop`.
+
+During that process, we also start to work with other frameworks and support them with that: [Loopback](https://github.com/PabloSzx/loopback-next/pull/1), [NestJS](https://github.com/PabloSzx/nestjs-graphql/pull/1), [Parse](https://github.com/parse-community/parse-server/pull/7394) and others.
```

Comment:
@PabloSzx will open these PRs, add descriptions there and then link it here.
Should we add logos to make it nicer?

Reply:
yes I'm saying once @PabloSzx will open them publicly (I think he is back tomorrow?) then we'll link them there with logos?